### PR TITLE
Updated Itemgroups

### PR DESCRIPTION
**Pull request by tarunsamanta2k20**
Updated Itemgroups

#640

### CODE REVIEW

File: src/main/java/be/uantwerpen/minelabs/item/ItemGroups.java [hunk 1]
```diff
@@ -44,21 +44,183 @@ public class ItemGroups {
...
+                stack.add(new ItemStack(Items.CALCIUM_CHLORIDE_DUST));
+                stack.add(new ItemStack(Items.SODIUM_CARBONATE_DUST));
+                stack.add(new ItemStack(Items.BORAX_DUST));
+                stack.add(new ItemStack(Items.COPPER_SULFATE_DUST));
+                stack.add(new ItemStack(Items.BORIC_ACID));
+                stack.add(new ItemStack(Items.COPPER_CHLORIDE_DUST));
+                stack.add(new ItemStack(Items.POTASSIUM_SULFATE_DUST));
+                stack.add(new ItemStack(Items.POTASSIUM_NITRATE_DUST));
+                stack.add(new ItemStack(Items.POTASSIUM_CHLORIDE_DUST));
+                stack.add(new ItemStack(Items.POTASSIUM_SULFATE_DUST));
```
Comment (pixar02) [inline src/main/java/be/uantwerpen/minelabs/item/ItemGroups.java:70]: `POTASSIUM_SULFATE_DUST` was added twice.
Please remove 1

File: src/main/java/be/uantwerpen/minelabs/item/ItemGroups.java [hunk 2]
```diff
@@ -44,21 +44,183 @@ public class ItemGroups {
...
+                stack.add(new ItemStack(Items.SODIUM_ATOM));
+                stack.add(new ItemStack(Items.MAGNESIUM_ATOM));
+                stack.add(new ItemStack(Items.ALUMINIUM_ATOM));
+                stack.add(new ItemStack(Items.SILICON_ATOM));
+                stack.add(new ItemStack(Items.PHOSPHORUS_ATOM));
+                stack.add(new ItemStack(Items.SULFUR_ATOM));
+                stack.add(new ItemStack(Items.CHLORINE_ATOM));
+                stack.add(new ItemStack(Items.ARGON_ATOM));
+                stack.add(new ItemStack(Items.POTASSIUM_ATOM));
+                stack.add(new ItemStack(Items.CADMIUM_ATOM));
```
Comment (pixar02) [inline src/main/java/be/uantwerpen/minelabs/item/ItemGroups.java:150]: Supposed to be Calcium not Cadmium.
Causes duplicate Cadmium

File: src/main/java/be/uantwerpen/minelabs/item/ItemGroups.java [hunk 3]
```diff
@@ -44,21 +44,183 @@ public class ItemGroups {
...
+                stack.add(new ItemStack(Items.TITANIUM_ATOM));
+                stack.add(new ItemStack(Items.MAGNESIUM_ATOM));
```
Comment (pixar02) [inline src/main/java/be/uantwerpen/minelabs/item/ItemGroups.java:152]: Supposed to be Manganese not Magnesium. Causes duplicate Magnesium

File: src/main/java/be/uantwerpen/minelabs/item/ItemGroups.java [hunk 4]
```diff
@@ -44,21 +44,183 @@ public class ItemGroups {
...
+                stack.add(new ItemStack(Items.PHOTON_QUANTUMFIELD));
+                stack.add(new ItemStack(Items.NEUTRINO_QUANTUMFIELD));
+                stack.add(new ItemStack(Items.WEAK_BOSON_QUANTUMFIELD));
+                stack.add(new ItemStack(Items.UPQUARK_QUANTUMFIELD));
+                stack.add(new ItemStack(Items.DOWNQUARK_QUANTUMFIELD));
+                stack.add(new ItemStack(Items.GLUON_QUANTUMFIELD));
+                stack.add(new ItemStack(Items.ELECTRON_QUANTUMFIELD));
+                stack.add(new ItemStack(Items.PHOTON_QUANTUMFIELD));
+                stack.add(new ItemStack(Items.NEUTRINO_QUANTUMFIELD));
+                stack.add(new ItemStack(Items.WEAK_BOSON_QUANTUMFIELD));
```
Comment (pixar02) [inline src/main/java/be/uantwerpen/minelabs/item/ItemGroups.java:222]: Duplicate entries.
Adding them once is enough.

**Review by pixar02 — CHANGES_REQUESTED** [review]

File: src/main/java/be/uantwerpen/minelabs/item/ItemGroups.java (hunk 4 above)
Comment (tarunsamanta2k20) [reply src/main/java/be/uantwerpen/minelabs/item/ItemGroups.java:222]: @pixar02 i know, I saw that someone did mologram that way.

**Review by pixar02 — APPROVED** [review]

**Comment by tarunsamanta2k20** [comment]
@pixar02 please add me as a member of your organization. I want to contribute more in the future.

**Pull request merged**